### DISCUSSION TITLE
배송비 관련 문의 드립니다.

### DISCUSSION THREAD

**Discussion by piaoyujing** [commerce-api-naver/commerce-solutions#83] [묻고 답하기]
안녕하세요.
사용하는 API는 주문 > 상품 주문 정보 구조체   입니다.
1. productOrder > deliveryFeeAmount 는 배송비 합계  이 항목에 대해서 상세설명 부탁드립니다.
2. 만약 한개 장바구니에 상품이 여러개이고 상품마다 배송비가 다 다를때
상품별 배송비는 어떤필드로 알수 있을가요?
3. 만약 한  상품에서 또 옵션두개를 함께 구입하였다면 이 상품의 배송비를 어떻게 알수 있을가요?

**Comment by commerce-api-naver** [comment]
안녕하세요. 커머스API 이민석입니다.

혹시 개발중이신 솔루션의 이름 혹은 솔루션 ID를 알려주실 수 있을까요?
몇가지 기본적인 설정 확인 후 솔루션 개발에 도움이 될만한 정보와 함께 가이드 드리도록 하겠습니다.

**Comment by piaoyujing** [comment]
솔루션 이름은 유일입니다.

**Reply by commerce-api-naver** [reply]
@piaoyujing 님, 커머스API 이민석입니다.

커머스솔루션 등록 기준으로 다시 한 번 확인해주실 수 있을까요?
이 공간은 커머스솔루션 개발사를 위한 공간입니다. 등록하신 솔루션 명칭 기준 혹은 솔루션 ID를 알려주셔도 좋습니다.
